core/src/main/java/io/fabric8/maven/core/service/ApplyService.java

```diff
@@ -1071,6 +1075,9 @@ public boolean applyProject(Project project) {
      * Returns true if the ProjectRequest is created
      */
     public boolean applyProjectRequest(ProjectRequest entity) {
+        // Check whether project creation attempted before
+        if (projectsCreated.contains(getName(entity)))
+            return true;
```

Comment:
According to the method signature, I think the return value should be false if projectsCreated contains getName(entity)

Reply:
Nice catch! Thanks :+1: